io/pom.xml

```diff
@@ -21,7 +21,10 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
```

Comment:
This should be test scope, shouldn't it?

Reply:
Sure.